vhost-device-scmi/src/scmi.rs

```diff
@@ -247,6 +259,7 @@ pub const SENSOR_AXIS_DESCRIPTION_GET: MessageId = 0x7;
 pub const SENSOR_CONFIG_GET: MessageId = 0x9;
 pub const SENSOR_CONFIG_SET: MessageId = 0xA;
 pub const SENSOR_CONTINUOUS_UPDATE_NOTIFY: MessageId = 0xB;
+pub const SENSOR_UPDATE: MessageId = 0x1;
```

Comment:
I would add the following comment above this line to avoid confusion about the message id numbers:
``// Notification message ids:``

Reply:
I add it in next push